java/src/com/github/bazel_contrib/contrib_rules_jvm/junit5/Java17SystemExitToggle.java

```diff
@@ -56,10 +59,15 @@ public void allow() {
   private Unsafe getUnsafe() throws ReflectiveOperationException {
     // We want to avoid opening java.lang, so let's jump through hoops
 
-    // First problem: we can't call `Unsafe.getUnsafe` directly.
-    var c = Class.forName("sun.misc.Unsafe");
-    var lookup = MethodHandles.privateLookupIn(c, MethodHandles.lookup());
-    var handle = lookup.findStaticVarHandle(c, "theUnsafe", c);
-    return (Unsafe) handle.get();
+    // First problem: we can't call `Unsafe.getUnsafe` directly and it might also be called
+    // differently.
+    Class<?> unsafe = Class.forName("sun.misc.Unsafe");
```

Comment:
The way this was written before allowed us to avoid needing to open any more java modules. Please revert to the original implementation.

Reply:
Just tried this with Java 19 and it appears to work as expected.